### PR TITLE
fix: show correct values on preference setting view

### LINKED ISSUE #440: [BUG] preference input item validate nubmer type error
**描述你的问题（Describe the bug）**

image

seem that the [`schemaProvider.validate(preferenceName, value)`](https://github.com/ide-framework/core/blob/27980ba2f8489bc69ce0423bba13758c65d0db78/packages/preferences/src/browser/preferenceItem.view.tsx#L254) validate string when the input is number

**复现路径（To Reproduce）**

**预期表现（Expected behavior）**

**问题截图（Screenshots）**

**环境信息（Environment）**

- OS: [e.g. macOS 11.2 Apple M1/Windows10/Windows11]
- OpenSumi Version [e.g. 2.13.0]

**补充信息（Additional context）**

### PR DESCRIPTION
**Pull request by erha19**
### Types

- [x] 🐛 Bug Fixes

### Background or solution

fixed #440, #473

also fix `fontWeight` configuration define.

### Changelog

show correct values on preference setting view

### CODE REVIEW

File: packages/components/src/select/index.tsx [hunk 1]
```diff
@@ -250,35 +253,82 @@ export function Select<T = string>({
   allowOptionsOverflow,
   dropdownRenderType = 'fixed',
   description,
-  showNotMatchAsWarning,
+  notMatchWarnning,
```
Comment (erha19) [inline packages/components/src/select/index.tsx:256]: 把原有 `showNotMatchAsWarning` 配置项修改成  `notMatchWarnning` 接受不匹配的报错信息了，否则国际化不好处理

Comment (bytemain) [reply packages/components/src/select/index.tsx:256]: 应该是 warning 吧？不是 warnning

**Comment by bytemain** [comment]
LGTM

File: packages/components/src/select/index.tsx (hunk 1 above)
Comment (erha19) [reply packages/components/src/select/index.tsx:256]: 你说的对

**Review by Aaaaash — APPROVED** [review]
LGTM

**Pull request merged**